playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -0,0 +1,42 @@
+env_id: "openshift"
+openstack_dns_domain: "example.com"
+openstack_nameservers: ["192.168.1.1"]
```

Comment:
JFYI, I have a WIP patch what addresses the hosts DNS setup as a persisted config, via the heat stack diven dns node IPs merged with custom vars. It helped to address my dev env blocker.
Will polish it and submit soon.